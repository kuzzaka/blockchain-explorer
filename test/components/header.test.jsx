import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';

test('Header snapshot', () => {
  const wrapper = shallow(<Header title="My Title" />);
  expect(wrapper).toMatchSnapshot();
});
