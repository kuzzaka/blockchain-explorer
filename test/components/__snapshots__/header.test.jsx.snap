// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<header>
  <h1>
    My Title
  </h1>
  <Connect(SearchBar) />
</header>
`;
